heisenbridge/private_room.py

```diff
@@ -270,8 +276,12 @@ def from_config(self, config: dict) -> None:
         if self.network_name is None and self.network_id is None:
             raise Exception("No network or network_id key in config for PrivateRoom")
 
+        if "hidden_room_id" in config:
+            self.hidden_room_id = config["hidden_room_id"]
+
     def to_config(self) -> dict:
         return {
+            "hidden_room_id": self.hidden_room_id,
```

Comment:
Don't store this in the config since it's a state value, fetch it during bridge initialization and set when creating a room.

Reply:
It's being used by the join function to determine if the room has a hidden room "attached".
So you mean it should instead fetch the join_rule during init, and read it from there?

Reply:
Yes, fetch the join_rule during init and fill that value from there instead of storing it twice.